### PR TITLE
Assimp: Fix compilation with GCC 12 and update to v5.2.4

### PR DESCRIPTION
**Pull request by SirLynix**
Since assimp compiles with -Werror, a GCC update breaks the compilation.
See https://github.com/assimp/assimp/issues/4433

### CODE REVIEW

File: packages/a/assimp/xmake.lua [hunk 1]
```diff
@@ -83,12 +83,16 @@ package("assimp")
...
+        if package:is_plat("linux", "macosx", "mingw") and package:version():le("v5.2.4") then
+            cxflags = "-Wno-array-bounds"
+        end
         if package:is_plat("mingw") and package:version():lt("v5.1.5") then
             -- CMAKE_COMPILER_IS_MINGW has been removed: https://github.com/assimp/assimp/pull/4311
             io.replace("CMakeLists.txt", "CMAKE_COMPILER_IS_MINGW", "MINGW", {plain = true})
         end
 
-        import("package.tools.cmake").install(package, configs)
+        import("package.tools.cmake").install(package, configs, {cxflags=cxflags})
```
Comment (waruqi) [inline packages/a/assimp/xmake.lua:99]: add space ` = `

Comment (waruqi) [inline packages/a/assimp/xmake.lua:99]: add space ` = `

**Comment by SirLynix** [comment]
Can this PR be merged? I need it to fix MinGW compilation on my CI.

**Comment by SirLynix** [comment]
Thanks!

**Pull request merged**